### ISSUE TITLE
Duplicate Getting Started page and remove Self-Onboarding option from new version

### ISSUE THREAD

**Issue by daniellex0** [hackforla/website#1080]
### Overview
We need to duplicate the [Getting Started page](https://www.hackforla.org/getting-started), and remove the self-onboarding option from one copy of the page, because the video link doesn't work. However we will eventually have the self-onboarding again, once the slideshow is finished, so we want to save the old version of the page.

### Action Items
- [x] Duplicate the Getting Started page- old version of the page should remain the same and just be renamed "Getting Started with Video." 
   - [x] This page should not be linked to anywhere at the moment - but we are saving it until we have the self-onboarding option again (so that we don't need to change the code back later)
- [x] Name the new duplicate of the page just "Getting Started." 
- [x] Remove the "self-onboarding" section (top left) of the new Getting Started page duplicate 
- [x] Center the Guided Onboarding section of the new Getting Started page duplicate
- [x] Ask Bonnie to remove the Self-Onboarding option language on all of the Meetup onboarding events

### Resources/Instructions
[Getting Started page](https://www.hackforla.org/getting-started)

<img width="450" alt="Screen Shot 2021-02-21 at 9 31 37 AM" src="https://user-images.githubusercontent.com/67400366/108633176-b4c09a80-7427-11eb-8239-1deaa1fd3920.png">
